test/e2e/specs/editor/various/style-variation.spec.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'adding blocks', () => {
+	test( 'Should switch to the plain style of the quote block', async ( {
+		page,
+		pageUtils,
+	} ) => {
+		await pageUtils.createNewPost();
+
+		// Inserting a quote block
+		await pageUtils.insertBlock( { name: 'core/quote' } );
+		await page.keyboard.type( 'Quote content' );
+
+		await pageUtils.clickBlockToolbarButton( 'Quote' );
+
+		const plainStyleButton = await page.locator(
+			'button[role="menuitem"]:has-text("Plain")'
+		);
+		await plainStyleButton.click();
+
+		// Check the content
+		const quoteContent = `<!-- wp:quote {"className":"is-style-plain"} -->\n<blockquote class="wp-block-quote is-style-plain"><p>Quote content</p></blockquote>\n<!-- /wp:quote -->`;
```

Comment:
Nitpick: let's inline this in `toBe()` and replace `\n` with actual line breaks for readability.

Reply:
I get this error
<img width="695" alt="CleanShot 2022-04-11 at 09 48 11@2x" src="https://user-images.githubusercontent.com/33403964/162689524-0257e915-4ac7-4f12-af46-2259f7354c1c.png">

when I replace it with line breaks.
<img width="734" alt="CleanShot 2022-04-11 at 09 48 18@2x" src="https://user-images.githubusercontent.com/33403964/162689560-9ce13904-f7b0-455e-979a-3f57710429c8.png">

Reply:
Yeah, we should remove the preceding spaces.